packages/react/src/components/UIShell/HeaderMenu.js

```diff
@@ -219,6 +219,8 @@ class HeaderMenu extends React.Component {
   };
 }
 
-export default React.forwardRef((props, ref) => {
-  return <HeaderMenu {...props} focusRef={ref} />;
-});
+export default (() => {
```

Comment:
Is there anyway we could switch the pattern here to:

```jsx
const Component = React.forwardRef((props, ref) => {
  //
});

Component.displayName;
export default Component;
```

Just want to avoid adding IIFE blocks to exports